src/main/java/uk/gov/hmcts/cmc/claimstore/services/FormaliseResponseAcceptanceService.java

```diff
@@ -82,6 +95,10 @@ private void createEventForReferToJudge(Claim claim, String authorisation) {
         CaseEvent caseEvent;
         if (isCompanyOrOrganisation(response.getDefendant())) {
             eventProducer.createRejectOrganisationPaymentPlanEvent(claim);
+            User user = userService.authenticateAnonymousCaseWorker();
+
+            PDF document = claimantResponseReceiptService.createPdf(claim);
+            documentService.uploadToDocumentManagement(document, user.getAuthorisation(), claim);
```

Comment:
this returns updated claim with the document. Please use updated claim further in the code otherwise the returned claim will be outdated

Reply:
None of the subsequent code that updates the claim will execute because if this code executes none of the subsequent code's preconditions will be met. @ckalista brought up a possible refactor of this code in the future.